### PR TITLE
ASM-3306 Changes to support powerconnect deployments

### PR DESCRIPTION
**Pull request by asaenz**
MERGE WITH: https://github.com/dell-asm/asm-deployer/pull/976

### CODE REVIEW

File: lib/puppet_x/dell_powerconnect/possible_facts/base.rb [hunk 1]
```diff
@@ -29,7 +29,8 @@ def self.register(base)
       cmd 'show bootvar'
     end
 
-    base.register_param 'systemmodelid' do
+    #TODO:  GO UPDATE THE JAVA SIDE TO LOOK FOR MODEL INSTEAD OF SYSTEMMODELID
```
Comment (gavin-scott) [inline lib/puppet_x/dell_powerconnect/possible_facts/base.rb]: What's the urgency of this TODO? I don't see a corresponding asm-core PR.

**Comment by gavin-scott** [comment]
+1 AFAIK except for potential need to update asm-core

@sushilrai do you want to review this also?

**Comment by sushilrai** [comment]
+1

**Comment by gavin-scott** [comment]
Going to merge, I think it's unlikely that any problem with powerconnect re the asm-core changes needed will cause any blocking issues.

File: lib/puppet_x/dell_powerconnect/possible_facts/base.rb (hunk 1 above)
Comment (asaenz) [inline lib/puppet_x/dell_powerconnect/possible_facts/base.rb]: @gavin-scott There wasn't any real urgency, it was mostly there so I wouldn't miss it, which I did unfortunately :( :-1: 

I don't THINK it should break anything on the Java side unless something is really relying on the device model. This is the line that needs to be changed.  It was mostly a small change I wanted to slip in just for some consistency in the fact names.  
https://github.com/dell-asm/asm-core/blob/master/webapps/ASMUI/src/main/java/com/dell/asm/ui/controller/DellSwitchController.java#L173

**Pull request merged**